aiidalab_widgets_base/wizard.py

```diff
@@ -0,0 +1,183 @@
+"""The wizard application allows the implication of a Wizard-like GUI.
+
+Authors:
+
+    * Carl Simon Adorf <simon.adorf@epfl.ch>
+"""
+from enum import Enum
+
+import traitlets
+import ipywidgets as ipw
+
+
+class WizardApp(ipw.VBox):
+    class State(Enum):
```

Comment:
Would it make sense to move this guy to `WizardAppStep`? This way we don't need to import both `WizardApp` and `WizardAppStep` when creating a new widget.

Reply:
Yes, makes more sense. Done.